Add buck rules in coreml llama transformer

YIWENX14 opened the pull request:
Summary: Add buck rules in coreml llama transformer for importing the modules to internal repo.

Differential Revision: D70415647

YIWENX14 commented:
@pytorchbot label "topic: not user facing"

cccclai commented on examples/apple/coreml/llama/TARGETS:
```diff
@@ -0,0 +1,66 @@
+        "llama_transformer.py",
+    ],
+    _is_external_target = True,
+    base_module = "executorch.examples.apple.coreml.llama",
```
Is it needed?

metascroy commented on pyproject.toml:
```diff
@@ -92,6 +92,7 @@ flatc = "executorch.data.bin:flatc"
 # TODO(mnachin T180504136): Do not put examples/models
 # into core pip packages. Refactor out the necessary utils
 # or core models files into a separate package.
+"executorch.examples.apple" = "examples/apple"
```
Can we make this more specific to examples/apple/coreml/llama?

metascroy commented:
@YIWENX14 lint check is failing.  Can you run the OSS linter?

From executorch folder
```
lintrunner -a
```

(If lintrunner is not already installed, it can be installed with `pip install -r requirements-lintrunner.txt` from executorch).

YIWENX14 committed:
Add buck rules in coreml llama transformer (#9017)

Summary:

Add buck rules in coreml llama transformer for importing the modules to internal repo.

Reviewed By: metascroy, billmguo

Differential Revision: D70415647
